File: Level1/220818/javascript/StrangeCharacters.test.js
import { solution } from './StrangeCharacters';

test('uppear', () => {
  expect(solution('try hello world')).toBe('TrY HeLlO WoRlD');
});

test('lower', () => {
  expect(solution('TRY HELLO WORLD')).toBe('TrY HeLlO WoRlD');
});

File: Programmers/Level1/220818/java/StrangeCharactersTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.*;

class StrangeCharactersTest {
  @Test
  void UpperCase() {
    StrangeCharacters strangeCharacters = new StrangeCharacters();

    assertEquals("TrY HeLlO WoRlD", strangeCharacters.solution("try hello world"));
    assertEquals("   A   A", strangeCharacters.solution("   A   a"));
  }

  @Test
  void LowerCase() {
    StrangeCharacters strangeCharacters = new StrangeCharacters();

    assertEquals("TrY HeLlO WoRlD", strangeCharacters.solution("tRy hElLo WOrLd"));
  }
}
